lib/rclex/resource_server.ex

```diff
@@ -301,6 +301,7 @@ defmodule Rclex.ResourceServer do
   @spec get_identifier(charlist(), charlist()) :: charlist()
   defp get_identifier(node_namespace, node_name) do
     if node_namespace != '' do
+      # FIXME: 'node name must not contain characters other than alphanumerics or '_'
       "#{node_namespace}/#{node_name}" |> String.to_charlist()
```

Comment:
namespace に '/' が入るため、以下のエラーが出るようです。

```
>>> [rcutils|error_handling.c:108] rcutils_set_error_state()
This error state is being overwritten:

  'node name must not contain characters other than alphanumerics or '_', at /tmp/binarydeb/ros-foxy-rcl-1.1.13/src/rcl/node.c:164'

with this new error message:

  'subscription's implementation is invalid, at /tmp/binarydeb/ros-foxy-rcl-1.1.13/src/rcl/subscription.c:462'

rcutils_reset_error() should be called after error handling to avoid this.
<<<
```

対応には namespace の命名規則の変更が必要です。

Reply:
あ、これ私のせいですね・・。リファクタリングで機能を纏めた際にnamespaceとnode_nameの扱いを纏めてしまったせいっぽいです。nifの方ではnamespaceの有無はちゃんと分けられているので、横着せずにnameとnsをタプル等で扱うのがよいかと。パパっと直せそうであればお願いできますか？ややこしそうなら置いておいてください。